telluric/georaster.py

```diff
@@ -529,13 +529,10 @@ def __del__(self):
     @classmethod
     def _raster_opener(cls, filename, *args, **kwargs):
         """Return handler to open rasters (rasterio.open)."""
-        with warnings.catch_warnings():
```

Comment:
does it mean we are going to have warnings now?

Reply:
Yes, or at least we are not discarding them here (too low level, IMO). Are you OK with it? Should we add filters somewhere else?